rna/serializers.py

```diff
@@ -22,20 +22,21 @@ def get_pk_field(self, model_field):
         Returns a default instance of the pk field, unlike
         the parent class, which omits the pk field.
         """
-        return self.get_field(model_field)
+        return self.get_fields().get(model_field)
 
 
 class UnmodifiedTimestampSerializer(serializers.ModelSerializer):
```

Comment:
The purpose of this class is to preserve created and modified timestamps provided by the server. Otherwise the sync process will fail. What errors were you getting before you commented it out?

Incidentally that's also the cause of the flake8 failure travis is complaining about: this is where `parse_datetime` was used.